jetty-ee8/jetty-ee8-websocket/jetty-ee8-websocket-javax-server/src/main/config/modules/ee8-websocket-javax.mod

```diff
@@ -19,9 +19,4 @@ lib/ee8-websocket/jetty-ee8-websocket-servlet-${jetty.version}.jar
 lib/ee8-websocket/jetty-javax-websocket-api-@jakarta.websocket.api.version@.jar
 lib/ee8-websocket/jetty-ee8-websocket-javax-client-${jetty.version}.jar
 lib/ee8-websocket/jetty-ee8-websocket-javax-common-${jetty.version}.jar
-lib/ee8-websocket/jetty-ee8-websocket-javax-server-${jetty.version}.jar
-
-[jpms]
-# The implementation needs to access method handles in
-# classes that are in the web application classloader.
-add-reads: org.eclipse.jetty.websocket.javax.common=ALL-UNNAMED
+lib/ee8-websocket/jetty-ee8-websocket-javax-server-${jetty.version}.jar
```

Comment:
If this line was missing, does this mean that we don't have a distribution test for ee8 javax websocket?

Reply:
> If this line was missing, does this mean that we don't have a distribution test for ee8 javax websocket?

which line was missing? I only removed the `jpms` lines here